bin/i18n-codeorg/out.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Give the script a verbose name / documentation, it's not clear to me what this does looking at it quickly